package.json

```diff
@@ -1,18 +1,18 @@
 {
   "name": "ethos-design-system",
-  "version": "1.1.10",
+  "version": "1.1.12",
   "description": "Ethos Design System v1",
   "main": "./src/components/index.js",
   "license": "UNLICENSED",
   "scripts": {
-    "build": "yarn css:build",
+    "build": "yarn css:build && yarn css:format",
     "css:build": "node-sass src/components/index.scss src/components/design-system.css",
     "css:dev": "chokidar src/**/*.scss -c 'yarn build' --initial",
+    "css:format": "prettier --write \"src/components/**/*.*css\"",
     "cy:open": "cypress open",
     "cy:run": "cypress run --browser chrome --config video=false",
     "e2e:ci": "start-server-and-test styleguide http://localhost:9008 cy:run",
     "e2e:dev": "start-server-and-test styleguide http://localhost:9008 cy:open",
-    "format": "prettier --write \"src/components/**/*.*css\"",
     "lint": "eslint src --max-warnings 0",
     "preversion": "yarn lint && yarn test && yarn build && yarn stats && git diff --exit-code",
     "spotcheck": "yarn build && echo \"\n*****\n* View the spotcheck at http://localhost:8080/spotcheck.html\n*****\n\" && webpack-dev-server ",
```

Comment:
We got rid of `release` that creates the tags somewhere else, huh? We probably should just remove `spotcheck` at some point (not related to this PR per se)

Reply:
It also looks like `spotcheck` is hosted at `http://localhost:8081/` vs. `http://localhost:8080/spotcheck.html`

Reply:
yeah, no one's running that script I don't believe haha